README.md

```diff
@@ -1,634 +1,1828 @@
 
-Due to the (potentially) large feature set, it may be the case that not all features can fit into a single script. If high-level language and low level plutus-core optimizations are not enough to address this, it may be necessary to split features across multiple versions of Cardano-Loans. Besides a mild inconvenience, there is no cost to doing this; credit history transcends any one version, and users may choose what version to use depending on what features they need.
+### On-Chain, Emergent Credit History
+
+Cardano-Loans automatically generates the credit history for every borrower; there is no additional
+cost for this. Every part of the loan is tracked, not just whether it was a default. For example,
+lenders can see how much of the loan a borrower paid back before they defaulted. Therefore, not all
+defaults (should) impact creditworthiness in the same way. Futhermore, when new versions of the
+protocol come out, the borrower's credit history is not lost; a borrower's credit history transcends
+the protocol's version (assuming they are using the same credential across versions).
+
+Consider what this means for the non-banked people who live in countries unable to support their own
+credit history databases due to lack of money, corruption, etc. By using Cardano-Loans, the
+non-banked can trustlessly start building their own credit histories for as little as 0.3 ADA per loan
+action. This likely results in a credit history database that is 1,000-10,000x cheaper to run than
+the credit history systems in developed nations like the United States. And despite being
+astronomically cheaper to maintain, it offers things not even the contemporary databases offer:
+
+- Identity theft is *cryptographically impossible*.
+- Credit histories are globally accessible and verifiable.
+
+Using Cardano-Loans, the non-banked can finally leap-frog passed the incompetence/corruption of
+their own nations and build up their own, (possibly) globally recognized, economic identity.
+
+### Negotiable Collateralization
+
+Since the lenders explicitly set the relative values of each collateral asset used, there is no need
+to use oracles or enable liquidations (these are not meant to be margin loans). However, there is a
+risk for the value of the collateral to fall precipitously during the duration of the loan. If the
+collateral ever falls too low in value, **this is the lender's fault**. The lender either should
+have asked for a higher relative rate, or a different asset all together. It is up to the lender to
+use proper risk-management.  If people want to take back control from governments, they need to
+accept more responsibility. *Freedom isn't free.*
+
+### Under-Collateralized Loans
```

Comment:
Somewhere in this section, can you add something... a small sub-heading, maybe... to explain how the elegant & persuasive explanations here would still hold true if volatility swings were to suddenly make a lot of properly collateralised loans undercollateralised... including any situations long-term lender "reputation" might be wrecked involuntarily by huge volatility swings?

Or, if this problem can't be avoided, explain why it is normal & should be considered an acceptable risk?  In either case I think this high-level overview would suit non-technical people (company execs, managers) considering this system vs. some more centralised alternatives.

Reply:
I am coming around to your way of thinking about not needing to continually price the collateral.  Instead shifting the burden to the lenders to properly price the risk.